app/pages/lab-pages-editor/components/TasksPage/TasksPage.jsx

```diff
@@ -101,7 +101,9 @@ export default function TasksPage() {
 
   function deleteTask(taskKey) {
     // First check: does the task exist?
-    if (!workflow || !taskKey || !workflow?.tasks?.[taskKey]) return;
+    // if (!workflow || !taskKey || !workflow?.tasks?.[taskKey]) return;
+    // UPDATE: DON'T actually check if the task exists.
+    // This is useful if a Step refers to a Task that doesn't exist.
```

Comment:
I was thinking of keeping this in as a warning, since other functions usually have this `if (!thing) return` safety pattern, but for this function specifically, that safety _breaks_ the code. Maybe I'll rewrite this to a simpler warning note so it doesn't look like commented-out code

Reply:
deleteTask() has now been updated:
```
  function deleteTask(taskKey) {
    if (!workflow) return;
    // NOTE: don't check if task actually exists before deleting it.
    // This is useful if a Step somehow refers to a Task that doesn't exist.
    ...
```

I've reintroduced the safety check for 'workflow', which I shouldn't have removed. Comment added to explain, yo, pls don't check for Task before trying to delete said `Task.taskKey === ''` is also valid, so the safety doesn't check for `if (!taskKey) return` either. 👌